makefiles/S3.toml

```diff
@@ -58,8 +65,20 @@ script = "${HOME}/.fluvio/bin/fluvio install --develop fluvio/fluvio-package"
 
 [tasks.bump-fluvio]
 dependencies = ["install-fluvio-package", "bump-fluvio-latest"]
-script = """${HOME}/.fluvio/bin/fluvio package --test bump dynamic ${FLUVIO_VERSION} """
+script = """
+${HOME}/.fluvio/extensions/fluvio-package \
+    ${FLUVIO_PUBLISH_TEST} \
+    bump \
+    dynamic \
+    ${FLUVIO_VERSION}
+"""
 
 [tasks.bump-fluvio-latest]
```

Comment:
Can we consolidate into a single `bump` task  instead of duplicating scripts?

Reply:
If you look closely, these are actually two separate commands. One is running

- `fluvio package bump latest VERSION+git`

and the other is running

- `fluvio package bump dynamic VERSION`

It would take a whole lot more conditionals and scripting in order to consolidate them into one task, this is much simpler.

Reply:
fair enough